src/uncontrast/pages/histogram.jsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import Plot from "react-plotly.js";
+
+const HistogramChart = () => {
+  var x1 = [];
+  var x2 = [];
+  var y1 = [];
+  var y2 = [];
+  for (var i = 1; i < 100; i++) {
+    let k = Math.random();
+    x1.push(k * 5);
+    x2.push(k * 10);
+    y1.push(k);
+    y2.push(k * 2);
+  }
+  console.log({ x1: x1, y1: y1 }, { x2, y2 });
+  var trace1 = {
```

Comment:
Why use `var`?

Reply:
its a code i pasted from docomentaition just to play with. i remove it now